ci/gpu/build.sh

```diff
@@ -39,30 +39,35 @@ export NUMPY_EXPERIMENTAL_ARRAY_FUNCTION=1
 # SETUP - Check environment
 ################################################################################
 
-logger "Check environment..."
+gpuci_logger "Check environment"
 env
 
-logger "Check GPU usage..."
+gpuci_logger "Check GPU usage"
 nvidia-smi
 
-logger "Activate conda env..."
-source activate gdf
-conda list
+gpuci_logger "Activate conda env"
+. /opt/conda/etc/profile.d/conda.sh
+conda activate rapids
+conda info
+conda config --show-sources
+conda list --show-channel-urls
```

Comment:
This block should be removed, its repeated below.

Reply:
complete